File: Modul 3. Kondisional/PRAK305/PRAK305-2310817120003-NatalieGraceKatiandagho.py
detik = int(input())

hari = int(detik / (24 * 3600))
jam = int((detik % (24 * 3600)) / 3600)
menit = int((detik % 3600) / 60)
detik2 = int(detik % 60)

if detik >= 0 and detik < 61 :
    print(f'00:00:{detik}')
elif detik >= 60 and detik < 3600 :
    print(f'00:{menit:02d}:{detik2:02d}')
elif detik >= 3600 and detik < 86400:
    print(f'{jam:02d}:{menit:02d}:{detik2:02d}')
else:
    print(f'{hari}hari {jam:02d}:{menit:02d}:{detik2:02d}')
 

File: Modul 3. Kondisional/PRAK305/PRAK305-2310817120003-NatalieGraceKatiandagho.c
#include <stdio.h>

void main()
{
    int detik;
    int detik2;
    int menit;
    int jam;
    int hari;

    scanf("%d", &detik);

    hari = detik / (24 * 3600);
    jam = (detik % (24 * 3600)) / 3600;
    menit = (detik % 3600) / 60;
    detik2 = detik % 60;

    if (detik >= 0 && detik < 61)
    {
        printf("00:00:%02d", detik);
    }
    else if (detik >= 60 && detik < 3600)
    {
        printf("00:%02d:%02d", menit, detik2);
    }
    else if (detik >= 3600 && detik < 86400)
    {
        printf("%02d:%02d:%02d", jam, menit, detik2);
    }
    else if (detik >= 86400)
    {
        printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik2);
    }
}
